build.gradle

```diff
@@ -469,84 +470,78 @@ task fakePublish {
+                x64Base = "${System.env.CODEBUILD_SRC_DIR_Linux_x64_Artifacts_FIPS}"
+                aarch64Base = "${System.env.CODEBUILD_SRC_DIR_Linux_aarch64_Artifacts_FIPS}"
+            } else {
+                artifactId = project.name
+            }
+            def x86Jar = "${x64Base}/${jarBaseName}"
+            def aarch64Jar = "${aarch64Base}/${jarBaseName}"
+            def sourceJar = "${x64Base}/AmazonCorrettoCryptoProvider-sources.jar"
+            def javadocJar = "${x64Base}/AmazonCorrettoCryptoProvider-javadoc.jar"
+            artifact source: file(x86Jar),classifier:"linux-x86_64",extension:"jar"
+            artifact source: file(aarch64Jar),classifier:"linux-aarch_64",extension:"jar"
+            artifact emptyJar
+            artifact source: file(javadocJar),classifier:"javadoc",extension:"jar"
+            artifact source: file(sourceJar),classifier:"sources",extension:"jar"
+
+            pom {
+                name = pomName
+                description = 'The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard JCA/JCE interfaces.'
+                url = 'https://github.com/corretto/amazon-corretto-crypto-provider'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
                     }
                 }
-            }
-        }
-        repositories {
-            maven {
-                url = 'https://aws.oss.sonatype.org/service/local/staging/deploy/maven2/'
-                credentials {
-                    username mavenUser
-                    password mavenPassword
+                scm {
+                    url = 'https://github.com/corretto/amazon-corretto-crypto-provider.git'
+                    connection = 'scm:git:git@github.com:corretto/amazon-corretto-crypto-provider.git'
+                    developerConnection = 'scm:git:git@github.com:corretto/amazon-corretto-crypto-provider.git'
+                }
+                developers {
+                    developer {
+                        id = 'amazonwebservices'
+                        organization = 'Amazon Web Services'
+                        organizationUrl = 'https://aws.amazon.com'
+                    }
                 }
             }
         }
     }
+}
 
-    signing {
-        sign publishing.publications.mavenJava
-    }
-} else {
-    task publish(overwrite: true) {
-        doFirst {
-            ant.fail('Insufficient configuration for publishing')
+nexusPublishing {
+    repositories {
+        sonatype {
+            //nexusUrl = uri('https://aws.oss.sonatype.org/service/local/staging/')
+            // This *should* be a snapshot only repo and thus safer to play with
+            nexusUrl = uri('https://aws.oss.sonatype.org/content/repositories/snapshots')
```

Comment:
Essentially yes. Some projects maintain snapshots which is the tip of development for people who want the latest and greatest options. ACCP doesn't do that. However, that means that the snapshot repo is:
* Not used by anyone currently to pull ACCP
* Intended for people grabbing absolute tip development anyway (and it might expire, I'm not positive)
* requires explicit opt-in to pull code from

So this is just an extra level of caution. If we accidentally were to release a version, it would go to a low-impact place. (I can also append "-SNAPSHOT" to the version, which makes this even safer.)